### ISSUE TITLE
Include analytics for Content Style Guide on dashboard

### ISSUE THREAD

**Issue by robertjmccarthy** [ministryofjustice/moj-frontend#1050]
## What
Track visits to the Content Style Guide pages on the MoJ Design System documentation site and include the data in the design system dashboard.

## Why
As we are trying to demonstrate usage of the design system, specifically reuse of building blocks, Content Style Guide entries form part of this. Whilst tracking usage of individual entries will not be possible, analytics showing the volume of visits will support this work.

We will refine and improve the metrics as we develop the dashboard further.

## Who needs to be involved
- Performance Analyst

## Done
When the design system dashboard includes page views from Google Analytics for the Content Style Guide pages.

**Comment by helennickols** [comment]
Measuring the style guide: GDS look at source traffic and have GA on the accordions open/close.

**Comment by Whersi** [comment]
@chrispymm @robertjmccarthy I'm planning to use Google Tag Manager to monitor the tracking for each accordion reveal in the Content Style Guide and also elements in the homepage.

However, we need to install a web container that you can review below;

[Google Tag Manager Web Container Guide](https://support.google.com/tagmanager/answer/14847097?sjid=17599064209798764845-EU)


Do let me know if this is a feasible solution to implement.

**Comment by Whersi** [comment]
The setup differs because Google Tag Manager (GTM) is a separate tool with its own account, different but compliments Google Analytics.  

I have created a GTM account and sent an admin invitation for your access.

**Comment by murrlipp** [comment]
Looks like there is something up with how our GTM is linked, as the GTM assistant isn't recognising the tag.

![Image](https://github.com/user-attachments/assets/f32c45f1-b168-450a-bd53-c10a93bf957c)

Do we use this for anything currently? Will starting from scratch cause any unintended consequences?

FYI @Whersi this ticket isn't assigned to @chrispymm but you've tagged him in a comment above.

**Comment by murrlipp** [comment]
@Whersi I followed [these steps](https://support.google.com/tagmanager/answer/14847097?hl=en&ref_topic=15191151&sjid=12958321187054196259-EU) and created [a branch/PR](url) where I've added the appropriate code for GTM into our template.

Once this is merged into main you should be able to follow [step 3](https://support.google.com/tagmanager/answer/14842872?hl=en&ref_topic=15191151&sjid=12958321187054196259-EU) and [step 4](https://support.google.com/tagmanager/answer/14842769?hl=en&ref_topic=15191151&sjid=12958321187054196259-EU) to define the tags and events we want to track.
